modules/ROOT/pages/using.adoc

```diff
@@ -95,6 +95,66 @@ $ crc setup # Initialize environment for cluster
 $ crc start # Start the cluster
 ----
 
+[id='about-pullsecrets']
+== About Pull Secrets
+While starting cluster configured with {openshift} or {ushift} preset {prod} would request you to provide a pull secret, it's required to pull some {ocp} specific container images.
+You need to download it from the Pull Secret section of the link:https://console.redhat.com/openshift/create/local[{prod} page on the {rh} Hybrid Cloud Console].
+
+Once you provide this pull secret, it gets stored in Operating System's credential manager so that {prod} doesn't ask for pull secret again in case you delete existing cluster and create
+a new one.
+
+Please note that this pull secret would only be removed from the Operating System's credential manager when user runs [command]`{bin} cleanup` command.
+
+If you manually want to remove/update pull secret from your Operating System's credential manager configuration.
+
+[id='clearning-credential-manager-pullsecret']
+=== Clearing Pull Secret from Credential Manager
+Steps to clear entries from the Credential Manager on different operating systems.
+
+==== Windows
+
+1. Open the Control Panel.
+2. Go to `User Accounts` > `Credential Manager`.
+3. Choose `Windows Credentials`.
+4. Find the {prod} pull secret entry you want to delete.
+5. Click on the entry to expand it.
+6. Click `Remove` to delete the credential.
```

Comment:
Asciidoc syntax for ordered lists. See https://docs.asciidoctor.org/asciidoc/latest/syntax-quick-reference/#lists

```suggestion
. Open the Control Panel.
. Go to `User Accounts` > `Credential Manager`.
. Choose `Windows Credentials`.
. Find the {prod} pull secret entry you want to delete.
. Click on the entry to expand it.
. Click `Remove` to delete the credential.
```